core/start-blueos-core

```diff
@@ -62,29 +62,29 @@ fi
 # From that 1min30s, the startup time is about ~25s, and originally, ~37s, meaning that the
 # remaining (~65 seconds) is the docker shutting down, and the Linux booting up.
 PRIORITY_SERVICES=(
-    'autopilot',"$SERVICES_PATH/ardupilot_manager/main.py"
+    'autopilot',"nice --19 $SERVICES_PATH/ardupilot_manager/main.py"
     'cable_guy',"$SERVICES_PATH/cable_guy/main.py"
-    'video',"mavlink-camera-manager --default-settings BlueROVUDP --mavlink tcpout:127.0.0.1:5777 --gst-feature-rank omxh264enc=0,v4l2h264enc=250,x264enc=260 --log-path /var/logs/blueos/services/mavlink-camera-manager --verbose"
+    'video',"nice --19 mavlink-camera-manager --default-settings BlueROVUDP --mavlink tcpout:127.0.0.1:5777 --gst-feature-rank omxh264enc=0,v4l2h264enc=250,x264enc=260 --log-path /var/logs/blueos/services/mavlink-camera-manager --verbose"
     'mavlink2rest',"mavlink2rest --connect=udpin:127.0.0.1:14000 --server 0.0.0.0:6040 --system-id $MAV_SYSTEM_ID --component-id $MAV_COMPONENT_ID_ONBOARD_COMPUTER4"
 )
 
 SERVICES=(
     'beacon',"$SERVICES_PATH/beacon/main.py"
-    'kraken',"$SERVICES_PATH/kraken/main.py"
-    'wifi',"$SERVICES_PATH/wifi/main.py --socket wlan0"
-    'bridget',"$SERVICES_PATH/bridget/main.py"
+    'kraken',"nice -19 $SERVICES_PATH/kraken/main.py"
```

Comment:
You probably want 19 and not -19, -19 is almost highest priority

Reply:
`-19` sets to 19, `--19` sets to `-19`

Reply:
yep, apparently I messed up nginx too

Reply:
wait, no, joao is right =P. but I still messed up nginx